### ISSUE TITLE
Weakly supervised problem

### ISSUE THREAD

**Issue by lq-111** [backtime92/CRAFT-Reimplementation#26]
Hi author, thanks for opening source your reimplementation about CRAFT. I hope to get the same results as claimed in the original paper, but now still have some problems (only 81+% performance on ICDAR15 dataset). Do you have any suggestions? I guess gaussian map generation or watershed may contain some problems.

If possible, could I contact you through email, wechat or so on for convenient communication?

**Comment by zhenx** [comment]
Agree. We might open a wechat group to discuss the problems we have, which is more efficient.

**Comment by backtime92** [comment]
@zhenx @lq-111 Ok，you can give me your weChat number, I will contact you.

**Comment by lq-111** [comment]
@backtime92 Thanks for your reply. My wechat number is lq_lalala .

**Comment by zhenx** [comment]
@backtime92 Thanks for your help. My wechat number is southernfish.

**Comment by Shualite** [comment]
@backtime92  I also care about this issue. My wechat number is ShuYang097. Thank you!

**Comment by zobeirraisi** [comment]
How many epochs are used for the training of Syndata or icdar15?

**Comment by xiongjialu1** [comment]
您好，我的微信号是：zyxzjswifexjl,可以拉我进群吗，谢谢您
